packages/kit/src/views/Onboardingv2/pages/ConnectYourDevice.tsx

```
@@ -1062,6 +1254,85 @@ const isSupportedDevice = (deviceType: string) => {
+      // Clear force transport type on QR wallet creation error
+      void backgroundApiProxy.serviceHardware.clearForceTransportType();
+      errorToastUtils.toastIfError(error);
+      void trackHardwareWalletConnection({
+        status: 'failure',
+        deviceType: EDeviceType.Pro,
+        isSoftwareWalletOnlyUser,
+        hardwareTransportType: 'QRCode',
+      });
+      throw error;
+    }
+  }, [createQrWallet, isSoftwareWalletOnlyUser, navigation]);
```

ESLint

[error] React Hook useCallback has a missing dependency: 'closePopover'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)